tests/test_watchmaker.py

```diff
@@ -19,43 +21,6 @@
     from mock import patch
 
 import watchmaker
-from watchmaker import static
-
-
-@pytest.fixture
-def watchmaker_arguments():
-    """Return default watchmaker arguments."""
-    watchmaker_arguments = {
-        'salt_states': 'highstate',
-        'no_reboot': False,
-        'admin_groups': None,
-        'computer_name': None,
-        'admin_users': None,
-        'log_level':
-        'debug',
-        'ou_path': None,
-        'config_path': None,
-        'environment': None,
-        'extra_arguments': [],
-        'log_dir': u'/var/log/watchmaker'
-    }
-
-    return watchmaker_arguments
-
-
-@pytest.fixture
-def watchmaker_client(watchmaker_arguments):
-    """Return  watchmaker client with defaults."""
-    return watchmaker.Client(watchmaker_arguments)
-
-
-@pytest.fixture
-def default_config():
-    """Return default configuration for watchmaker."""
-    config_path = str(resources.files(static) / 'config.yaml')
```

Comment:
I guess we weren't using this fixture, but the use of the `importlib_resources` library to resolve filepaths was very intentional. I am not sure if it is relevant to the use case here, though. Did you try using the resources interface?

In general, the resources interface is the "correct" way these days in Python to reference paths to static files in a python package, instead of trying to construct them from os.path or Pathlib. The resources interface works for _all_ package types, which the others do not (think zip packages and in-memory runtimes, such as when using pyoxidizer).

Reply:
I did not use the resources interface, I don't know of a case where I would need to use it.  In my tests there is a tests/resources folder that I do use os.join path on to build the path, but that is a different case than this correct?  I am not importing a package, simply referencing test resource files.

Reply:
Yeah I just wasn't sure. I don't think the tests are included in the watchmaker package, so maybe we don't have to worry about that particular concern.

Reply:
Yeah, the package directory is src, so tests are not included, which one would expect.